.gitignore

```diff
@@ -1,2 +1,6 @@
 .idea
-.terraform/
\ No newline at end of file
+**/.terraform/
+.terraform.lock.hcl
+terraform.tfstate
+terraform.tfstate.backup
```

Comment:
those files are part of `**/.terraform/`


you don't need those lines

Reply:
nope
![image](https://github.com/env0/k8s-modules/assets/83922349/0974e499-af5e-4a96-b201-fca711d92a0d)